tianshou/policy/modelfree/pg.py

```diff
@@ -190,18 +190,19 @@ def forward(
         """
         # TODO: rename? It's not really logits and there are particular
         #  assumptions about the order of the output and on distribution type
-        logits, hidden = self.actor(batch.obs, state=state, info=batch.info)
-        if isinstance(logits, tuple):
-            dist = self.dist_fn(*logits)
+        # See https://pytorch.org/docs/stable/distributions.html
+        action_dist_input, hidden = self.actor(batch.obs, state=state, info=batch.info)
+        if isinstance(action_dist_input, tuple):
+            dist = self.dist_fn(*action_dist_input)
         else:
-            dist = self.dist_fn(logits)
+            dist = self.dist_fn(action_dist_input)
 
         # in this case, the dist is unused!
         if self.deterministic_eval and not self.training:
-            act = self._get_deterministic_action(logits)
+            act = self._get_deterministic_action(action_dist_input)
         else:
             act = dist.sample()
-        result = Batch(logits=logits, act=act, state=hidden, dist=dist)
+        result = Batch(logits=action_dist_input, act=act, state=hidden, dist=dist)
```

Comment:
Yes, the DistBatchProtocol should be adjusted everywhere. Note that you then have to construct the Batch with
`action_dist_input=action_dist_input`, and you will need to scan the whole codebase for places where the logits attribute of a Batch is used (which can be through `.logits` or through `["logits"]`)

Reply:
So I assume you are fine with the renaming right?

Reply:
yes, action_dist_input sounds like the right name, thanks!